core/app/finders/spree/products/find.rb

```diff
@@ -16,7 +16,7 @@ def initialize(scope:, params:, current_currency:)
         @sort_by          = params.dig(:sort_by)
         @deleted          = params.dig(:filter, :show_deleted)
         @discontinued     = params.dig(:filter, :show_discontinued)
-        @properties       = params.dig(:product_properties)
+        @properties       = params.dig(:filter, :properties)
```

Comment:
How in general should we approach such changes to params structure? It seems like a breaking change to me (or maybe I'm wrong). Are we fine with it?

Reply:
Fortunately, this change is only a modification of the previous PR which was not part of any release yet, hence it will not break anything